app-backend/db/src/main/scala/TaskDao.scala

```diff
@@ -558,13 +558,19 @@ object TaskDao extends Dao[Task] {
 
   def countProjectTaskByStatus(
       projectId: UUID
-  ): ConnectionIO[Map[TaskStatus, Int]] = {
-    (fr"""
+  ): ConnectionIO[Map[TaskStatus, Int]] = (fr"""
     SELECT status, COUNT(id)
     FROM tasks
     WHERE annotation_project_id = ${projectId}
     GROUP BY status;
-    """).query[(TaskStatus, Int)].to[List].map(_.toMap)
+    """).query[(TaskStatus, Int)].to[List].map { list =>
+    List(
+      TaskStatus.Unlabeled,
+      TaskStatus.LabelingInProgress,
+      TaskStatus.Labeled,
+      TaskStatus.ValidationInProgress,
+      TaskStatus.Validated
+    ).fproduct(status => list.toMap.getOrElse(status, 0)).toMap
```

Comment:
:sunglasses: I didn't know about `fproduct`, that's cool

I think there's maybe a more efficient way to do this though, since Scala isn't lazy. If you pull `list.toMap` outside of the `fproduct`, it should get evaluated once instead of 5 times:

```suggestion
    """).query[(TaskStatus, Int)].to[List].map { list =>
    val statusMap = list.toMap
    List(
      TaskStatus.Unlabeled,
      TaskStatus.LabelingInProgress,
      TaskStatus.Labeled,
      TaskStatus.ValidationInProgress,
      TaskStatus.Validated
    ).fproduct(status => statusMap.getOrElse(status, 0)).toMap
```